shared/src/persistence/dynamsoft/getScannerInterface.test.js

```diff
@@ -261,7 +261,7 @@ describe('getScannerInterface', () => {
     global.window.document = {
       addEventListener: () => null,
       createElement: () => ({
-        cloneNode: function () {
+        cloneNode() {
```

Comment:
i think we're missing a colon here. 

```
        cloneNode: () {
```

Reply:
I think that's actually valid es6.  you can define function properties on an object like that

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects#defining_methods